packages/types/src/chains.ts

```diff
@@ -218,4 +218,75 @@ export interface Chain {
   };
 }
 
+export interface MemoKeys {
```

Comment:
I noticed that there's no file depending on these new types.

Reply:
Yep I just generated it using chain registry schemas, actually we can also generate this data they have some json for that